packages/shared/src/components/squads/SquadMemberItemAdditionalContent.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { ReactElement } from 'react';
+import { SourceMember, SourceMemberRole } from '../../graphql/sources';
+import { Button, ButtonSize } from '../buttons/Button';
+import BlockIcon from '../icons/Block';
+import MenuIcon from '../icons/Menu';
+import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { SquadMemberBadge } from './SquadMemberAuthorBadge';
+import { PromptOptions, usePrompt } from '../../hooks/usePrompt';
+import { UserShortInfo } from '../profile/UserShortInfo';
+import { ModalSize } from '../modals/common/types';
+
+interface SquadMemberActionsProps {
+  member: SourceMember;
+  onUnblock: React.MouseEventHandler;
+  onOptionsClick: React.MouseEventHandler;
+}
+
+function SquadMemberItemAdditionalContent({
+  member,
+  onUnblock,
+  onOptionsClick,
+}: SquadMemberActionsProps): ReactElement {
+  const { showPrompt } = usePrompt();
+  const { user: loggedUser } = useAuthContext();
+  const { role, user } = member;
+
+  const onConfirmUnblock = async (e: React.MouseEvent) => {
+    e.preventDefault();
+
+    const options: PromptOptions = {
```

Comment:
This can be outside the function (optional comment)

Reply:
I think we can keep this here, so JS won't have to create this object until necessary.